src/coreclr/src/jit/jitconfigvalues.h

```diff
@@ -202,6 +223,8 @@ CONFIG_INTEGER(EnableIncompleteISAClass, W("EnableIncompleteISAClass"), 0) // En
                                                                            // intrinsic classes
 #endif                                                                     // defined(DEBUG)
 
+CONFIG_INTEGER(JitAlignLoops, W("JitAlignLoops"), 1) // If set, align inner loops
```

Comment:
Yes, it is and I forgot to check with you offline on it. But I actually want to get rid of the flag present in `clrconfigvalues.h` because it is JIT related flag. If I remove it, will it need GUID change?

Reply:
No, it wouldn't need a GUID change (I think). It would be ok to remove it from clrconfigvalues.h if nobody outside the JIT uses it.

Reply:
Alright, I will remove the flag from VM.